draft-ietf-dprive-dnsoquic.md

```diff
@@ -230,7 +230,7 @@ see {{Section 4.2 of RFC8490}}.
 
 DoQ connections are established as described in the QUIC transport
 specification {{!RFC9000}}. During connection establishment, DoQ support is
-indicated by selecting the ALPN token "doq" in the crypto handshake.
+indicated by selecting the ALPN (Application-Layer Protocol Negotiation) token "doq" in the crypto handshake.
 
```

Comment:
The common practice would be text first, then acronym:
```
indicated by selecting the Application-Layer Protocol Negotiation (ALPN) token "doq" in the crypto handshake.
```

Reply:
Stewart Bryan has a related comment (see issue #146): _ALPN is not well known and should be expanded on first use. It would be useful to note that “doq” is a string. This is clarified below, but it is proposed
that that text be removed._

Reply:
Thanks - I did get that the wrong way round and should be fixed now. I agree about not needing to spell out it is a string which is why I didn't make that change.....